secondarylib/src/main/scala/com/socrata/datacoordinator/secondary/SecondaryWatcher.scala

```diff
@@ -175,24 +182,82 @@ class SecondaryWatcher[CT, CV](universe: => Managed[SecondaryWatcher.UniverseTyp
   }
 }
 
+object SecondaryWatcherClaimManager {
+  private val log = LoggerFactory.getLogger(classOf[SecondaryWatcherClaimManager])
+
+  // in-memory list of datasets that are actively being worked on by _this_ instance
+  private val whitelist = scala.collection.mutable.Set.empty[Long]
```

Comment:
Would it be clearer to rename whitelist as "currentlyWorkingOn" or something similar but shorter?